### ISSUE TITLE
Add the Noun Project API to moulinette - game icons

### ISSUE THREAD

**Issue by CyberFoxar** [SvenWerlen/moulinette-core#10]
Another source suggestion, this time for SVG and icons. I'd like to see [the noun project](https://thenounproject.com/) added to game icons. Their goal is to provide a wide variety of free icons for personal use.
They of course provide a free (if limited) API. I think this would complement nicely game-icons, just as an alternative. It requires the user to create an account and API key, though.

[Here's](https://thenounproject.com/) the developper portal, where you can know a bit more.

(also of note, their icons do not have the `width` and `height` property, like those from game-icons)

**Comment by SvenWerlen** [comment]
I played with the noun project and their API. It would be possible to implement this integration but it would require each individual to register for a playground plan (https://thenounproject.com/developers/) or pay 150$/mo. During the creation process, you'll also have to describe your project and I fear that this will bring attention to Moulinette and this type of integration would be considered against their policy (abusing of the free tier). I decided to not implement this for now. Thanks for your suggestion!
